lib/rex/exploitation/cmdstager/echo.rb

```diff
@@ -26,10 +26,14 @@ def initialize(exe)
   # and initialize opts[:enc_format].
   #
   def generate(opts = {})
-    opts[:temp] = opts[:temp] || '/tmp/'
-    opts[:temp].gsub!(/\\/, "/")
-    opts[:temp] = opts[:temp].shellescape
-    opts[:temp] << '/' if opts[:temp][-1,1] != '/'
+    if opts[:temp] == false
```

Comment:
@jvazquez-r7 great ... I will send you the needed pcap and screenshot after your final cleanup